starters/deno-oak-denodb/docker-compose.yml

```diff
@@ -12,6 +12,21 @@ services:
       - POSTGRES_DB=deno_db
     volumes:
       - 'deno_db_data:/var/lib/postgresql/data/'
+      
+  redis:
+    image: 'redis:alpine'
+
+    command: redis-server
+
+    ports:
+      - '6379:6379'
+
+    volumes:
+      - $PWD/redis-data:/var/lib/redis
```

Comment:
I've read that `$PWD` won't work on a windows machine, I'd suggest using `./redis-data:/var/lib/redis` instead.